azurerm/internal/services/loadbalancer/loadbalancer_resource.go

```diff
@@ -352,13 +386,50 @@ func expandAzureRmLoadBalancerFrontendIpConfigurations(d *pluginsdk.ResourceData
 		}
 
 		if v := data["subnet_id"].(string); v != "" {
+			subnetSet = true
+			properties.PrivateIPAddressVersion = network.IPVersionIPv4
+			if v := data["private_ip_address_version"].(string); v != "" {
+				properties.PrivateIPAddressVersion = network.IPVersion(v)
+			}
 			properties.Subnet = &network.Subnet{
 				ID: &v,
 			}
 		}
 
 		name := data["name"].(string)
-		zones := azure.ExpandZones(data["zones"].([]interface{}))
+		zones := &[]string{"1", "2"}
```

Comment:
The acceptance tests are failing with `frontendIPConfigurations has 2 zones specified. Only one zone can be specified for this resource.` So we should only set a single zone here?
```suggestion
		zones := &[]string{"2"}
```